object.go

```diff
@@ -199,3 +199,10 @@ func (uID UniqueID) String() string {
 	}
 	return fmt.Sprintf("%d", uID.Number)
 }
+
+// Verification documented here: https://developers.notion.com/reference/page-property-values#verification
+type Verification struct {
+	State      string      `json:"state"` // "verified" or "unverified"
```

Comment:
May be it's better to make a set of constant like this?

```go
type VerificationState = string

const (
    VerificationStateVerified   VerificationState = "verified"
    VerificationStateUnverified VerificationState = "unverified"
)
```

WDYT?

Reply:
makes sense, done